python/pylibcudf/pylibcudf/strings/combine.pyx

```diff
@@ -148,7 +148,6 @@ cpdef Column join_strings(Column input, Scalar separator, Scalar narep):
 
     return Column.from_libcudf(move(c_result))
 
-
```

Comment:
revert

Reply:
I think we've been doing two new lines between functions in implementation files